### PR TITLE
Added the Buttons and alerts

### PR DESCRIPTION
**Pull request by Rohitprasad83**
# Different types of alerts

alert

# Alert with close button
chrome_nwe27zmun2

# Different types of buttons
chrome_jIzxrLeAVe

### CODE REVIEW

File: alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+@import url(../styles.css);
+
+html {
```
Comment (Naveenchandar) [inline alert/alert.css:3]: Suggestion:
If you're using common style file, you can even move these styles to common file.

File: buttons/buttons.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    padding: 2rem;
+    margin: 1rem;
+}
+
+.btn {
+    padding: 1rem;
+    border-radius: 5px;
+    border: none;
+    cursor: pointer;
+    margin: 5px;
```
Comment (Naveenchandar) [inline buttons/buttons.css]: Please avoid using 'px' for spacing

File: buttons/buttons.css [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+.btn__dark {
+    background-color: var(--dark);
+    color: var(--white);
+}
+
+.btn__link {
+    background-color: #fff;
+    color: blue;
```
Comment (Naveenchandar) [inline buttons/buttons.css]: Suggestion: 
Please use hex code for consistency.

File: alert/alert.css (hunk 1 above)
Comment (Rohitprasad83) [reply alert/alert.css:3]: I am thinking of doing this after creating all of the components and then putting the common CSS to style.css

**Review by Naveenchandar — APPROVED** [review]
Looks good.

**Review by maitrakhatri — APPROVED** [review]
looks perfect to me

File: alert/alert.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
```
Comment (smilegupta) [inline alert/alert.html]: Can include more meta tags here

File: buttons/buttons.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Buttons</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+        integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA=="
```
Comment (smilegupta) [inline buttons/buttons.html]: inculde these font-awesome to the common css file

File: styles.css [hunk 6]
```diff
@@ -1,8 +1,18 @@
 :root {
-    --danger: #a50104;
-    --primary: #3590f3;
-    --success: #389446;
+    --danger: #ff002b;
+    --primary: #3a86ff;
+    --success: #3bc14a;
     --offline: #c9c9c9;
```
Comment (smilegupta) [inline styles.css:7]: This variable name can be imporved

**Review by smilegupta — COMMENTED** [review]
Just one suggestion... 
You can indent your code before pushing to git. this habit will help you in longer run

File: buttons/buttons.html [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+</head>
+
+<body>
+    <div class="container">
+        <button type="button" class="btn btn__primary">Primary</button>
+        <button type="button" class="btn btn__secondary">Secondary</button>
+        <button type="button" class="btn btn__disabled" disabled>Disabled</button>
+
+        <button type="button" class="btn btn__success">Success</button>
```
Comment (rohanmathur91) [inline buttons/buttons.html:20]: remove the blank line.

**Comment by Rohitprasad83** [comment]
> Just one suggestion... You can indent your code before pushing to git. this habit will help you in longer run

File: styles.css (hunk 6 above)
Comment (Rohitprasad83) [reply styles.css:7]: changed the danger variable to error

File: buttons/buttons.html (hunk 5 above)
Comment (Rohitprasad83) [reply buttons/buttons.html]: added the link of font-awesome to style.css

File: alert/alert.html (hunk 4 above)
Comment (Rohitprasad83) [reply alert/alert.html]: added one more meta tag for better accessibility

**Pull request merged**